tests/order/order.helpers.js

```diff
@@ -0,0 +1,686 @@
+const { gql } = require('@apollo/client');
+
+const createColor = async (color, operations) => {
+  const createdColor = await operations.mutate({
+    mutation: gql`
+      mutation($color: ColorInput!) {
+        addColor(data: $color) {
+          ... on Color {
+            _id
+          }
+          ... on Error {
+            message
+            statusCode
+          }
+        }
+      }
+    `,
+    variables: { color },
+  });
+
+  return createdColor.data.addColor._id;
+};
+const createMaterial = async (material, operations) => {
```

Comment:
I think it would be better to move this helper func in separate file in material folder